.gitignore

```diff
@@ -42,3 +42,13 @@ node_modules/
 
 # VS Code
 .vscode
+
+# Rockstor build
```

Comment:
I added these because I ended up working directly in the VM with VSCode remote dev, but this change can be separated out to it's own PR if you would prefer.

Reply:
@StephenBrown2 These changes to .gitignore are a nice addition actually - and yes I think it would be good to have them split out into their own issue/pull request set. We will then have them to point to for folks that want to add similar dev environment additions/enhancements such as these. It will also help with cherry picking these changes, in isolation, if they end up being required in another branch.

Thanks for thinking of this and highlighting it. Plus it will help to split out 'things' from one another. Always nice.
It can also help with reviewing to have stuff split down into more manageable chunks.